### PR TITLE
 #225 - bug fix , App crash on adding new trip keeping fields blank

### LINKED ISSUE #225: Fix bug in Add New Trip Screen
There is a bug in the Add New trip screen where the user enters the details of the trip.
- Keep everything blank and click on the "add trip" button directly.
- The app crashes

### PR DESCRIPTION
**Pull request by f71uday**
# Description

Added condition that Trip info cant be left blank
if fields left blank Snakbar generates

Fixes #225 

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [ ] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: Android/app/src/main/java/io/github/project_travel_mate/mytrips/AddNewTripActivity.java [hunk 1]
```diff
@@ -279,6 +283,17 @@ public void onClick(View view) {
             // Add a new trip
             case R.id.ok:
                 mTripname = tname.getText().toString();
+                Log.d("TRIP INFO" , "tripname" + mTripname);
+                Log.d("TRIP INFO" , "tripcity" + cityname.getText());
+                Log.d("TRIP INFO" , "tripdate" + tripStartDate.getText());
+
+                if (mTripname.equals("")) {
+                    Snackbar.make(mLinearLayout , "Trip name can't be left Blank" , Snackbar.LENGTH_LONG).show();
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/mytrips/AddNewTripActivity.java]: add constant strings to `strings.xml` file & use those strings here.

File: Android/app/src/main/java/io/github/project_travel_mate/mytrips/AddNewTripActivity.java [hunk 2]
```diff
@@ -279,6 +283,17 @@ public void onClick(View view) {
             // Add a new trip
             case R.id.ok:
                 mTripname = tname.getText().toString();
+                Log.d("TRIP INFO" , "tripname" + mTripname);
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/mytrips/AddNewTripActivity.java]: You can remove this log

**Comment by Swati4star** [comment]
I am facing one problem with this:

start typing trip name. Click on `Add Trip`. Nothing is happening.

(What's happening is snackbar is being displayed behind the keyboard & as the keyboard is open,user is not able to view that snackbar. You should hide the keyboard when `Add Trip` is pressed.

**Comment by f71uday** [comment]
okk i will get it done

**Comment by Swati4star** [comment]
@f71uday You can use this function to hide the keyboard : https://github.com/project-travel-mate/Travel-Mate/blob/59004203fe01da11f4c80f49cde655cd20fce74f/Android/app/src/main/java/utils/Utils.java#L57

**Comment by Swati4star** [comment]
@f71uday Any updates?

**Comment by f71uday** [comment]
Yes i started woking today . I was on a holiday break.

**Pull request closed without merging**